src/server/test/unit/liquidTagGrammer.test.ts

```diff
@@ -0,0 +1,299 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/*!
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ */
+
+import { expect } from 'chai';
+import * as nearley from 'nearley';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const grammar = require('../../Parser/liquidTagGrammar');
+
+describe('LiquidGrammar', () => {
+
+  it('test invalid attribute(entityform) in Liquid Expression', () => {
+    try
+    {
+     parseLiquidTag("{% entityform class: 'test' %}");
+    }
+    catch(err)
+    { 
+      expect(err.message).eq('Error parsing the output');   
+    }
+   });
+
+  it('test valid tag(entityform) in Liquid Expression', () => {
+    const parsedTag = parseLiquidTag("{% entityform name: 'test' %}");
+      expect(parsedTag).eq('entityform');
+   });
+ 
+   it('test valid index value(entityform with attribute name) in Liquid Expression', () => {
+       const parserOutput = parseLiquidAndGetKeyValue("{% entityform name: 'test' %}");
+       expect(parserOutput.size).eq(1);
+       const liquidExpKeys = parserOutput.keys();
+       expect(liquidExpKeys.next().value).eq('21');
+       expect(parserOutput.get('21')).eq('name');
+    });
+
+    it('test valid index value(entityform with attribute ID) in Liquid Expression', () => {
```

Comment:
indenting is off, yes, I know, I'm an annoying nit picker. Use VSCode's auto formatting via Ctr+Shift+P, then "Format Selection" or "Format Document"
![image](https://user-images.githubusercontent.com/3200210/128943381-d3174bca-0ec6-4254-bf1b-1c63d80c6b87.png)

 (you are using VSCode to edit, right :-) )

Reply:
done. from format document.